nxrm-azure-ha-helm/Chart.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v2
```

Comment:
missed license header

Reply:
I don't see license in the AWS ones. What've we've done here is to include a [LICENSE](https://github.com/sonatype/nxrm3-preview-ha-repository/blob/main/nxrm-aws-ha-helm/LICENSE) file

Reply:
This one https://github.com/sonatype/nxrm3-preview-ha-repository/blob/main/sample-aws-ha-yamls/aws-ha-external-dns-rbac.yaml for example. As far as I know, each file should contain license header